docs/user_guide/disconnected/index.md

```diff
@@ -4,286 +4,52 @@ title: Disconnected
 sidebar: sidebars/documentation.html
 ---
 
-# Katello Disconnected Utility
+# Katello Inter-Server Sync
 
 ## Intro
 
-If you are working in an environment where you wish to synchronize content from Red Hat's CDN and make available to your Katello servers that do not have a connection to the internet we have a utility that assists with this, known as katello-disconnected.  A diagram of the content flow is as follows:
+If you are working in an air-gapped network environment where some of your
+Katello servers do not have Internet connectivity, you may be interested in
+using the Katello Inter-Server Sync feature. This allows you to export repos
```

Comment:
Typically you want to define the acronym the first chance you use the full wording, e.g. Katello Inter-Server Sync (ISS)